crml/nft/src/lib.rs

```diff
@@ -152,6 +164,34 @@ decl_error! {
 		AddToUniqueIssue,
 		/// Tokens with different individual royalties cannot be sold together
 		RoyaltiesProtection,
+		/// The mass_drop activation_time or limit is invalid
+		MassDropInvalid,
+		/// There is no mass_drop associated with the series
+		NoMassDropExists,
+		/// There are no accounts in the whitelist
+		WhitelistEmpty,
+		/// The account_id isn't in the whitelist
+		NoAccountId,
+		/// The presale is invalid
+		PresaleInvalid,
+		/// There is no presale associated with the mass_drop
+		NoPresaleExists,
+		/// The quantity specified is higher than the mass_drop purchase limit
+		PurchaseQuantityTooHigh,
+		/// The mass_drop hasn't started yet
+		MassDropNotStarted,
+		/// The presale hasn't started yet
+		PresaleNotStarted,
+		/// There are not enough tokens left to purchase this amount
+		NotEnoughTokens,
+		/// The activation time has passed or is invalid
+		ActivationTimeInvalid,
+		/// This account is not registered in the whitelist
+		NotInWhitelist,
+		/// No owner exists for this collection id
+		NoCollectionOwner,
+		/// Can't mint additional when a mass drop is active
+		MassDropActive,
```

Comment:
consider changing this so it's not confused with activation time
```suggestion
		MassDropConfigured,
```